code/modules/projectiles/guns/projectile/automatic/bren.dm

```diff
@@ -1,7 +1,6 @@
 /obj/item/gun/projectile/automatic/bren
-	name = "\"Kalashnabren\" light machinegun"
-	desc = "A mutilated Kalash model that has been jury rigged to load its magazines from the top. Why, you may ask? Only god knows. No serial numbers nor publishing logos can even be found on the rifle.\
-		The trigger mechanism is attached by a cord externally to create large bursts of fire. Despite this thing looking like it might explode at any second, it can really shoot fast!"
+	name = "\"Bren\" light machine gun"
+	desc = "A select-fire top-loaded light machine gun chambered in 6.5mm. At a closer inspection, it seems to be the gruesome result of mental illness and an AK-pattern rifle."
```

Comment:
Funny but might need some information on fire rate.

Reply:
I'll catch this on rewrite.